Fix linker error in dead code (doesnt happen with LTO) when enable_ppapi=false

--- src/electron/BUILD.gn.orig	2022-10-20 18:35:58.433896013 +0200
+++ src/electron/BUILD.gn	2022-10-27 10:17:35.131568660 +0200
@@ -628,6 +628,10 @@
       "//ppapi/proxy",
       "//ppapi/shared_impl",
     ]
+    sources += [
+      "shell/renderer/electron_renderer_pepper_host_factory.cc",
+      "shell/renderer/electron_renderer_pepper_host_factory.h",
+    ]
   }
 
   if (enable_run_as_node) {
--- src/electron/filenames.gni.orig	2022-10-20 18:35:58.497928015 +0200
+++ src/electron/filenames.gni	2022-10-27 10:17:06.415830980 +0200
@@ -671,8 +671,6 @@
     "shell/renderer/electron_render_frame_observer.h",
     "shell/renderer/electron_renderer_client.cc",
     "shell/renderer/electron_renderer_client.h",
-    "shell/renderer/electron_renderer_pepper_host_factory.cc",
-    "shell/renderer/electron_renderer_pepper_host_factory.h",
     "shell/renderer/electron_sandboxed_renderer_client.cc",
     "shell/renderer/electron_sandboxed_renderer_client.h",
     "shell/renderer/guest_view_container.cc",
